src/postgres/tree/PostgreSQLTableTreeItem.ts

```diff
@@ -0,0 +1,49 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as _ from 'underscore';
```

Comment:
Remove this unused import. I thought we had lint rules that would automatically do this for you...I might look into that.